sessions/MachineLearning-100/101/101.001/2.md

```diff
@@ -0,0 +1,36 @@
+# What is machine learning used for?
+
+## Where machine learning shines
+
+- When it's not possible to define an algorithm to do a thing
+- When you want to personalise a thing
+- When you need to scale a thing
+
+
+## Questions that machine learning can answer
+
+“What is this thing?” - Classification
+- “Is this a spam email?”
+- “What plant is this?”
+- "Is this a picture of a dog or a cat?"
+- "Is this customer likely to churn?"
```

Comment:
"churn" is a bit jargonny; I wonder if it's worth spelling it out, something like "leave to go to one of our competitors"?